internal/zinc-persist/src/main/mima-filters/1.0.0.backwards.excludes

```diff
@@ -0,0 +1,8 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("sbt.internal.inc.binary.BinaryAnalysisFormat.readAPIs")
```

Comment:
Just an appreciation: this means that we **do not** provide bincompat for readers and writers, **only for the protobuf format**.